packages/components/addon/components/hds/pagination-bar/index.hbs

```diff
@@ -1,4 +1,73 @@
 {{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  <div class="hds-pagination-bar__total-count">1–10 of 100</div>
+
+  <nav class="hds-pagination-bar__pagination">
+    <ul class="hds-pagination-bar__page-list">
+      <li class="hds-pagination__page hds-pagination__page--previous">
+        <a class="hds-pagination__page-link" aria-label="Previous page" href="/page-?">
+          <FlightIcon @name="chevron-left" class="hds-pagination__nav-icon" /><span
+            class="hds-pagination__page-label"
+          >Previous</span>
+        </a>
+      </li>
+      <li class="hds-pagination__page hds-pagination__page--current">
+        <a class="hds-pagination__page-link" href="/page-1">1</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-2">2</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-3">3</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-4">4</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-5">5</a>
+      </li>
+      <li class="hds-pagination__page">
+        <span class="hds-pagination__page-link hds-pagination__page-link--ellipsis">...</span>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-10">10</a>
+      </li>
+      <li class="hds-pagination__page hds-pagination__page--next">
+        <a class="hds-pagination__page-link" href="/page-?" aria-label="Next page">
+          <span class="hds-pagination__page-label">Next</span><FlightIcon
+            @name="chevron-right"
+            class="hds-pagination__nav-icon"
+          />
+        </a>
+      </li>
+    </ul>
+  </nav>
+
+  <div class="hds-pagination-bar__page-size">
+    <Hds::Form::Select::Field class="hds-pagination-bar__page-size-select" as |F|>
+      <F.Label>Items per page</F.Label>
+      <F.Options>
+        <option value="Kubernetes">10</option>
+        <option value="Other">30</option>
+        <option value="Other">50</option>
+      </F.Options>
+    </Hds::Form::Select::Field>
+  </div>
+</div>
+
+{{!
+  TEMP ROUGH NOTES
+  ------------------
+  Still figuring out class naming and DOM structure...
+
+  WIP Structure (some of these will be separated out as sub-components, others just represent HTML/CSS structure):
+    * pagination-bar
+      * total-count
+      * pagination (separate out below children or keep grouped together in one list?)
+        * page-prev
+        * pages
+          * page
+            * page-link (some content aren't links though...)
+        * page-next
+      * page-size
```

Comment:
I'm not sure about some of the naming and what makes the most sense. Also not sure if I should keep the Previous and Next links in the same list with the number links. I may move them outside it later on...

Reply:
I think you probably only have to name things if you're going to make them separate components, yeah? 

Thinking it through, you probably wouldn't need to name the `<a>` element- it'll be within an `<li>` -> `<ul>` -> `<nav>` and if you've named all of those, that link will be really tightly scoped.

Reply:
Thanks, my preference is to not add class names to everything for styling like I did here but it seemed like that was the team preference? So thats why I did. Normally I would probably target nested elements by tag such as `<a>` and `<li>` for something like this.

I may break this out into sub-components but probably only 3 at the most for the main three modules within the PaginationBar. (Still considering what would make sense and will work on that in a new PR for the next step.)

Reply:
Let's bring it up in the eng sync, because I didn't do classes all the way down for the table component. I think we should use them when it makes sense (and mostly so far it's made sense, but there are a few exceptions).

Reply:
@alex-ju @didoo do either of you have any input? Thanks!

Reply:
Personally, putting classes on everything makes things seem cluttered, difficult to read, and seems like overkill (just my opinion, no offense). I'm happy to follow whatever the preferred approach is though.

Reply:
I went ahead and moved the previous and next links outside the list of page numbers btw. (They aren't quite the same as the numbered links so I don't think they need to be part of the list semantically and also it makes it easier to hide and show the number list for the compact variant of Pagination)